src/NATS.Client.JetStream/NatsJSManageConsumers.cs

```diff
@@ -0,0 +1,54 @@
+using NATS.Client.JetStream.Models;
+
+namespace NATS.Client.JetStream;
+
+public class NatsJSManageConsumers
+{
+    private readonly NatsJSContext _context;
+
+    public NatsJSManageConsumers(NatsJSContext context) => _context = context;
+
+    public ValueTask<ConsumerInfo> CreateAsync(
+        string stream,
+        string consumer,
+        ConsumerConfigurationAckPolicy ackPolicy = ConsumerConfigurationAckPolicy.@explicit,
+        CancellationToken cancellationToken = default) =>
+        CreateAsync(
```

Comment:
It was convenient for the tests and I thought it might be for actual usage too since creating a consumer with sensible defaults is somewhat challenging.

The reason `ConsumerConfigurationAckPolicy` is an argument is because enum itself defaults to `none` and `ack_policy=none` is not a valid option for pull consumers.